README.md

```diff
@@ -6,7 +6,9 @@ See [PEP 732](https://peps.python.org/pep-0732/).
 
 ## Updates
 
-These updates provide high level information about the Editorial Board's activities.
+We have a "changelog" of decisions: [CHANGELOG.md](CHANGELOG.md).
```

Comment:
Hmm I think the `Updates` section should list our monthly notes.
I would suggest just adding a new header/section for Changelog

eg 

```
## Changelog

We have a "changelog" of decisions: [CHANGELOG.md](CHANGELOG.md). 
```

Reply:
done